### DISCUSSION TITLE
Etcd behaviour when PVC hangs / Stuck / inaccessible

### DISCUSSION THREAD

**Discussion by IamSatyaonline** [etcd-io/etcd#14813] [General]
Hi,

We are running some of test cases to check the robustness of ETCD service. We have one of the test case for the same and need help. What will be the behaviour of Pods when PVC hangs / Stuck / inaccessible ?
If something goes wrong with PVC , will it impact the ETCD service if yes, what will be that impact ?

Thanks,
Satya

**Comment by ptabor** [comment]
It's a question to some deployment of etcd on k8s - not strictly etcd. 

1. As long as underlying disk (volume) will be available the etcd should operate. 
2. PVC is mostly about control plane operations on disks: attaching / deattaching... so the problem happens when 2 things are combined: failure of PVC and failure of an instance, such that another cannot be provisioned. 

It's better to host etcd on VMs directly rather than hosting it on K8s.
